scrapy/core/downloader/tls.py

```diff
@@ -0,0 +1,16 @@
+from OpenSSL import SSL
+
+
+METHOD_SSLv3 = 'SSLv3'
+METHOD_TLS = 'TLS'
+METHOD_TLSv10 = 'TLSv1.0'
+METHOD_TLSv11 = 'TLSv1.1'
+METHOD_TLSv12 = 'TLSv1.2'
+
+openssl_methods = {
+    METHOD_TLS:    SSL.SSLv23_METHOD,                   # protocol negotiation (recommended)
```

Comment:
Where did this name came from? It is a surprising to have METHOD_TLS doing negotiation and falling back to SSL.